src/Types/Filtering.hs

```diff
@@ -1,10 +1,15 @@
 module Types.Filtering where
 
 import Control.Exception
+import Control.Monad.State
 import Data.Typeable
 import Text.Printf
 import Data.List (intercalate)
 
+defaultTimeoutMicro = 1 * 10^6 :: Int
+defaultNumChecks = 10 :: Int
+defaultEvalLength = 100 :: Int
```

Comment:
What does length refer to here? Perhaps to clarify you should consider a more descriptive name.